src/app/dashboard/models/page.tsx

```
@@ -11,69 +11,45 @@ import UserDropdown from "~/components/user-dropdown";
 function ModelCard({
   model,
   onToggle,
+  usage,
 }: {
-  model: any;
+  model: ModelConfig;
   onToggle: () => void;
+  usage?: {
+    messagesUsed: number;
+    charactersUsed: number;
+    limits: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)